installer/ProjectReunionInstall.sln

```diff
@@ -24,6 +29,16 @@ Global
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{E6E59B30-9F55-4550-AA73-3B3B3DC89872}.Debug_test|Any CPU.ActiveCfg = Release|x64
+		{E6E59B30-9F55-4550-AA73-3B3B3DC89872}.Debug_test|Any CPU.Build.0 = Release|x64
+		{E6E59B30-9F55-4550-AA73-3B3B3DC89872}.Debug_test|ARM.ActiveCfg = Release|x64
```

Comment:
What's the point of Debug_test? Debug+Release I get but why Debug_test rather than just Debug?

Reply:
I don't know, I didn't hand-edit that, it's something VS populated. Should it be removed/altered? How so?

Reply:
VS automagically adding it due to something something vcxproj something?

Do you have any C# in your solution?

@BenJKuhn @kennykerr  @Scottj1s @aeloros can you explain this?